rosette/src/CommandLine.cc

```diff
@@ -105,6 +107,7 @@ int ParseCommandLine(int argc, char** argv) {
         /* Flags */
         {"verbose", no_argument, NULL, 'v'},
         {"quiet", no_argument, NULL, 'q'},
+        {"defer-lookup", no_argument, NULL, 'l'},
```

Comment:
In the end, what we want is the command-line interface to have a sort of `--compile` flag that just compiles and writes bytecode to the output file (possibly stdout). This doesn't look like it does that, so what's actually happening when you call it with `--defer-lookup`?

Reply:
I discussed this with @pyrocto yesterday.  And that is indeed the plan. To do that, we will need to create something that can serialize to disk the code and data, and be able to read a pre-compiled segment back in and execute it, whether it originates with Rosette or Rhoscala. This is just a step towards that end goal.